src/actions/google/drive/google_drive.ts

```diff
@@ -36,18 +38,33 @@ export class GoogleDriveAction extends Hub.OAuthAction {
 
       const filename = request.formParams.filename || request.suggestedFilename()
       if (!filename) {
+        const error: Error = {
+          http_code: HTTP_ERROR.bad_request.code,
+          status_code: HTTP_ERROR.bad_request.status,
+          message: `${HTTP_ERROR.bad_request.description} Error creating filename from request`,
+          location: "ActionContainer",
+          documentation_url: "TODO",
+        }
         resp.success = false
-        resp.message = "Error creating filename"
-        winston.error("Error creating filename")
+        resp.error = error
+        resp.message = error.message
+        resp.webhookId = request.webhookId
+        winston.error(`${error.message}`, {error, webhookId: request.webhookId})
         return resp
       }
       try {
         await this.sendData(filename, request, drive)
         resp.success = true
       } catch (e: any) {
+        const error: Error = errorWith(
+            HTTP_ERROR.internal,
+            "Error while sending data " + e.message,
```

Comment:
Do you know if google drive sends back any additional info besides `e.message`? It would be really nice if they also sent back a error code so we aren't erroring with just a 500. 

For example GCS error contains a `e.code` and `e.reason` that could be helpful for us to narrow down what the error was.

Reply:
https://github.com/looker-open-source/actions/blob/master/src/actions/google/gcs/google_cloud_storage.ts#L114

Reply:
Unsure, but added a clause to handle this.